docs/standard/glossary.md

```diff
@@ -0,0 +1,228 @@
+---
+title: .NET Glossary
+description: Find out the meaning of selected terms used in the .NET documentation.
+keywords: .NET glossary, .NET dictionary, .NET terminology, .NET platform, .NET framework, .NET runtime
+author: tdykstra
+ms.author: tdykstra
+ms.date: 07/08/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# .NET Glossary
+
+The primary goal of this glossary is to clarify meanings of selected terms and acronyms that appear frequently in the .NET documentation without definitions. Some of these terms are used inconsistently, so a secondary goal is to help reduce the confusion that inconsistent usage causes.
```

Comment:
> Why not fix the inconsistent usage? Don't you simply mean that sometimes the context determines what a word/phrase/acronym means?

There are too many articles with too many inconsistent usages to fix them all on anything like a reasonable schedule.  The doc set is too big compared to the writer resources to execute a task like that.

> imply that MS inadvertently uses inconsistent terminology

That's precisely the case

That said, I think we can drop the second sentence and just call out problematic usage in the individual entries.